python/lsst/fgcmcal/fgcmBuildStarsBase.py

```diff
@@ -516,15 +516,15 @@ def _fillVisitCatalog(self, visitCat, groupedDataRefs, bkgDataRefDict=None,
 
         Parameters
         ----------
-        visitCat: `afw.table.BaseCatalog`
-           Catalog with schema from _makeFgcmVisitSchema()
-        groupedDataRefs: `dict`
-           Dictionary with visit keys, and `list`s of
-           `lsst.daf.persistence.ButlerDataRef`
-        visitCatDataRef: `lsst.daf.persistence.ButlerDataRef`, optional
-           Dataref to write visitCat for checkpoints
-        bkgDataRefDict: `dict`, optional
-           Dictionary of gen3 dataRefHandles for background info. FIXME
+        visitCat : `afw.table.BaseCatalog`
+            Catalog with schema from _makeFgcmVisitSchema()
```

Comment:
End with period.   I'm not totally clear on this...is it just the schema that is needed, or is it that the schemas must match? (pre-existing, so feel free to ignore!)

Reply:
I've changed this to `Visit catalog.  See _makeFgcmVisitSchema() for schema definition.`